stl_to_obj/textures.py

```diff
@@ -0,0 +1,57 @@
+import json
+
+# filenames of all the PBR textures included in game in the textures folder, stored as a dict for each file
+TEXTURE_FOLDER = "textures"
+
+NORMAL_OPTIONS = "normal_enabled = true\nnormal_scale = -1.0\n"
+DEPTH_OPTIONS = "depth_enabled = true\ndepth_scale = -0.01\ndepth_deep_parallax = false\ndepth_flip_tangent = false\ndepth_flip_binormal = false\n"
+
+class Texture:
+    def __init__(self, folder, jpg_dict, uv1_scale = ""):
+        self.folder = folder
+        self.jpg_dict = jpg_dict
+        # uv shouldn't be in the dict because it doesn't need an ext_resource like the .jpgs
+        self.uv1_scale = uv1_scale 
+
+# dict of all the textures we already have
+texture_dict = dict()
+other_textures = dict()
+
+# add textures from a .json file to dict 
+# allows contributors to use their own textures (that we don't have)
+def add_textures(filename):
+    with open(filename, 'r') as f:
+        # load JSON object as dictionary
+        data = json.load(f)
+
```

Comment:
I think everything after this can be unindented - the file is loaded in the single line above and `f` is no longer needed.